src/material-examples/chips-autocomplete/chips-autocomplete-example.html

```diff
@@ -0,0 +1,28 @@
+<mat-form-field class="demo-chip-list">
+  <mat-chip-list #chipList>
+    <mat-chip
+      *ngFor="let fruit of fruits"
+      [selectable]="selectable"
+      [removable]="removable"
+      (remove)="remove(fruit)">
+      {{fruit.name}}
+      <mat-icon matChipRemove *ngIf="removable">cancel</mat-icon>
+    </mat-chip>
+    <input
+      placeholder="New fruit..."
+      matInput
```

Comment:
I don't think it needs the `matInput`, since the `mat-chip-list` should be acting as the `MatFormFieldControl`